src/modules/file.ts

```diff
@@ -15,7 +15,7 @@ export default class File extends API {
    * @returns Raw data
    */
   async content(hash: string) {
-    const response = await this.sendGet(`file/${hash}/data`)
+    const response = await this.sendGet(`file/${hash}/content`)
```

Comment:
Nice and easy. @balupton, this is going to come up with other redirects, which I like having available, as it means we can be a little more flexible with releasing updates to js-http-client. Any idea what the best way is to more permanently fix these CORS stoppers on the Go side of things?

Reply:
I'm not sure, as any redirect is a mark of deprecation, so we should coordinate immediate parch releases of this api client whenever we do such things with go-tectile. Patch releases will be adopted automatically by consumers of this client. This client is also the only one affected by cors.